src/modelfree/score_agent.py

```diff
@@ -59,7 +54,7 @@ def default_score_config():
     agent_b_type = 'zoo'   # type supported by policy_loader.py
     agent_b_path = '2'     # path or other unique identifier
     num_env = 1            # number of environments to run in parallel
-    episodes = 1           # number of episodes to evaluate
+    episodes = 20           # number of episodes to evaluate
```

Comment:
Good but the test cases for `score_agent` will now run much slower. Can you change the default there to 1? (Don't add it to every single test case, just check if `episodes` is present in the test config and if not set it to 1.)